db/migrate/20210709042135_create_external_upload_stubs.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class CreateExternalUploadStubs < ActiveRecord::Migration[6.1]
+  def change
+    create_table :external_upload_stubs do |t|
+      t.string :key, null: false
+      t.string :original_filename, null: false
+      t.integer :status, default: 1, null: false, index: true
+      t.uuid :unique_identifier, null: false
+      t.integer :created_by_id, null: false, index: true
+      t.string :upload_type, null: false
+
+      t.timestamps
+    end
+
+    add_index :external_upload_stubs, [:unique_identifier], unique: true
```

Comment:
Sorry I don't think we need a unique index here, the odds of a UUID collision should be quite low.

Reply:
See https://en.wikipedia.org/wiki/Universally_unique_identifier#Collisions for more information :)

Reply:
@tgxworld yes I know UUID collisions are extremely rare I just thought we would want a unique index too. All good!